serverjs/cards.js

```diff
@@ -1,5 +1,6 @@
 const fs = require('fs');
-var util = require('./util.js');
+const util = require('./util.js');
+const cardutil = require('../dist/util/Card.js');
```

Comment:
this line is causing tests to fail, is this directory not visible when testing? is it a casing issue?